contrib/platform/test/com/sun/jna/platform/win32/User32WindowMessagesTest.java

```diff
@@ -314,7 +314,7 @@ public boolean callback(HWND hWnd, Pointer data) {
 
             char[] windowText = new char[512];
             assertCallSucceeded("GetClassName", User32.INSTANCE.GetClassName(hWnd, windowText, windowText.length) != 0);
-            String className = Native.toString(windowText);
+            String className = W32StringUtil.toString(windowText);
```

Comment:
This pretends, that the binding is ok, but it is broken. The black magic conversion in W32StringUtil will not be done by a normal user, so `GetClassName(HWND, char[], int)` is just broken with `w32.ascii`.

Reply:
Yes it does, but that was the directions I got earlier that I should not change any JNA API unless absolutely necessary.